src/taskgraph/run-task/run-task

```diff
@@ -1033,6 +1033,33 @@ def _display_python_version():
         b"setup", b"Python version: %s\n" % platform.python_version().encode("utf-8")
     )
 
+def _display_subprocess_python_version():
+    try:
+        output = subprocess.check_output(
+            ["uv", "python", "list", "--only-installed"],
+        ).strip()    
+
+        if output:
+            for line in output.splitlines():
+                print_line(b"setup", b"Subprocess python version: " + line + b"\n")
+
+    except FileNotFoundError:
+
+        python_versions = ["python","python3"]
+
+        for python_executable in python_versions:
+            try:
+                version = subprocess.check_output(
```

Comment:
This isn't wrong, but just fyi, you should try and use `subprocess.run` instead of all the various `call` / `check_call` / `check_output` / `Popen` / whatever methods.

`subprocess.run` is designed to replace all of the above and is the officially recommended method unless you're doing something advanced (in which case using `Popen` is needed).

In this case the equivalend to `check_output` would be:
```
subprocess.run([...], check=True, capture_output=True)
```